en/frames_multicopter/holybro_x500v2_pixhawk6c.md

```diff
@@ -0,0 +1,159 @@
+# Holybro X500 V2 + Pixhawk 6C (PX4 Dev Kit)
+
+This topic provides full instructions for building the [Holybro X500 V2 ARF Kit](http://shop.holybro.com/x500-v2-kit_p1288.html), also known as the Holybro PX4 Dev Kit.
+
+![The fully built vehicle with props removed](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/kit_no_props.jpg)
+
+
+## Assembly
+
+### Holders
```

Comment:
What is a Holder?

Reply:
> What is a Holder?

That would be the payload and battery holder assembly.

Reply:
I've updated title to reflect that